convert.go

```diff
@@ -192,6 +192,20 @@ func settingFromValue(name string, description string, v reflect.Value) (Setting
 		sv := reflect.Indirect(reflect.ValueOf(s))
 		sv.FieldByName("Int64Value").Set(v.Addr())
 		return s, nil
+	case reflect.Map:
+		vTypeStored := v.Type()
+		if vTypeStored.String() == "map[string][]string" {
```

Comment:
suggestion: To keep with the style of the rest of the code and account for additional map types, this should use a `switch` rather than an `if`.